lib/screens/department_screen/department_screen.dart

```diff
@@ -0,0 +1,214 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:tsec_app/screens/department_screen/widgets/about_section.dart';
+import 'package:tsec_app/screens/department_screen/widgets/faculty_details_section.dart';
+import 'package:tsec_app/utils/themes.dart';
+
+import 'department_constants.dart';
+import 'widgets/department_screen_app_bar.dart';
+import 'widgets/drop_down_menu_item.dart';
+
+class DepartmentScreen extends StatefulWidget {
+  final String departmentName;
+
+  final String selectedItem;
+
+  const DepartmentScreen(
+      {Key? key,
+      required this.departmentName,
+      this.selectedItem = ABOUT_SECTION})
+      : super(key: key);
+
+  @override
+  _DepartmentScreenState createState() => _DepartmentScreenState();
+}
+
+class _DepartmentScreenState extends State<DepartmentScreen> {
+  late String selectedItem;
```

Comment:
It would be better if we use enum instead of string here

Reply:
I would like to know why 🙈

Reply:
If there are any future additions it will be easier to handle and IMO it is more readable than strings.

Also you just need a list of names instead of a constants file for that.